src/renderer/entities/network/model/network-model.ts

```diff
@@ -39,6 +40,53 @@ const $apis = createStore<Record<ChainId, ApiPromise>>({});
 const $connections = createStore<Record<ChainId, Connection>>({});
 const $connectionStatuses = createStore<Record<ChainId, ConnectionStatus>>({});
 
+// not sure where this should live
+// network-utils did not seem like the best place
```

Comment:
@tuul-wq what you you think?

Reply:
`features/network/NetworksList/lib/networks-list-utils.ts`